google/cloud/security/enforcer/batch_enforcer.py

```diff
@@ -18,6 +18,7 @@
 from __future__ import division
 from __future__ import print_function
 
+import threading
```

Comment:
nit: threading is part of the standard library, so it should be with __future__ section.